socorro/external/boto/connection_context.py

```diff
@@ -112,13 +182,11 @@ def _get_credentials(self):
         raise NotImplementedError
 
     #--------------------------------------------------------------------------
-    @staticmethod
-    def build_key(prefix, name_of_thing, id, version='v1'):
```

Comment:
`.build_keys()` returns a list, so I think if we were to do this, it'd return the first item of what `.build_keys()` returned.

Regardless, I looked through the codebase and the only thing that uses `.build_key()` is the classes defined in this directory plus the tests that test it. I couldn't think of a compelling reason to have `.build_key()` other than backwards compatibility. Given that, I decided it was better to remove it altogether.